packages/core/test/UnirepState/reputationProofEvent.test.ts

```diff
@@ -96,29 +89,6 @@ describe('Reputation proof events in Unirep State', function () {
         })
     })
 
-    describe('Init Unirep State', async () => {
```

Comment:
I think we should move `Init Unirep State` test suit to `InitUnirep.test.ts`(new one)